eth_definitions/download.py

```
@@ -365,6 +371,164 @@ def _build_solana_token(complex_token: dict[str, Any]) -> SolanaToken | None:
+        if "tests" in path.parts:
+            continue
+
+        try:
+            # Scan every file so the unsupported-features log covers the whole
+            # registry, regardless of which providers are enabled below.
+            records = load_display_formats(path, unsupported=unsupported)
+        except UnsupportedFeature as e:
+            # File skipped — its features were already collected into `unsupported`.
+            logging.info(f"skipping {path.relative_to(ROOT_DIR)} — {e}")
+            continue
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)